.circleci/cimodel/data/simple/util/branch_filters.py

```diff
@@ -4,6 +4,11 @@
     r"/release\/.*/",
 ]
 
+PR_BRANCH_LIST = [
```

Comment:
Removed the "filter out not PR runs" command from the job spec "pytorch_android_gradle_custom_build_single" (it's copied from the existing job spec "pytorch_android_gradle_build-x86_32", which is also updated) and use this new filter instead.

I checked existing data points generated by the other PR-only job: https://fburl.com/scuba/pytorch_binary_size/5u5wakvu - seems the 'gh/.../head' and 'pull/....' are the two main patterns for pull requests. I'm not 100% sure, though.